### PR TITLE
Shank 393 mock services

### PR DESCRIPTION
**Pull request by bschofield-va**
Mock services available as standalone application.
Port 8666
```run-local -m s```
```http://localhost:8666/help for available queries```

### CODE REVIEW

File: facilities-mock-services/src/main/resources/va-arcgis-health-facilities.json [hunk 1]
```diff
@@ -0,0 +1 @@
+NO REPSONSE
```
Comment (jblefkove-va) [inline facilities-mock-services/src/main/resources/va-arcgis-health-facilities.json]: ?

Comment (bschofield-va) [reply facilities-mock-services/src/main/resources/va-arcgis-health-facilities.json]: oh yeah... been trying to cag in so I could get a real response...

File: facilities-mock-services/real-depedency-service-requests [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#Wait times and satisfaction scores:
+snatch "access-to-care-wait-times" "https://www.accesstocare.va.gov/atcapis/v1.1/patientwaittimes"
+snatch "access-to-care-satisfaction" "https://www.accesstopwt.va.gov/Shep/getRawData?location=*"
+
+#474 benefits centers, 180 cemetaries, 320 vet centers:
+snatch "arcgis-benefits-facilities" "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VBA_Facilities/FeatureServer/0/query?f=json&inSR=4326&outSR=4326&orderByFields=Facility_Number&outFields=*&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1"
+
+snatch "arcgis-cemetaries" "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer/0/query?f=json&inSR=4326&orderByFields=SITE_ID&outFields=*&outSR=4326&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1"
+
+snatch "arcgis-health-facilities" "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VHA_VetCenters/FeatureServer/0/query?f=json&inSR=4326&orderByFields=stationno&outFields=*&outSR=4326&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1"
```
Comment (jblefkove-va) [inline facilities-mock-services/real-depedency-service-requests]: these are vet centers

File: facilities-mock-services/real-dependency-service-requests [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+snatch "arcgis-cemetaries" "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer/0/query?f=json&inSR=4326&orderByFields=SITE_ID&outFields=*&outSR=4326&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1"
+
+snatch "arcgis-health-facilities" "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VHA_VetCenters/FeatureServer/0/query?f=json&inSR=4326&orderByFields=stationno&outFields=*&outSR=4326&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1"
+
+#1289 health facilities:
+snatch "va-arcgis-vet-center-facilities" "https://gis.va.gov/server/rest/services/VA/FacilitySitePoint_VHA/FeatureServer/0/query?f=json&inSR=4326&outSR=4326&orderByFields=Sta_No&outFields=*&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=s_abbr!=%27VTCR%27%20AND%20s_abbr!=%27MVCTR%27"
```
Comment (jblefkove-va) [inline facilities-mock-services/real-dependency-service-requests]: These are backwards... line 27 is vet centers, line 30 is health facilities

Comment (bschofield-va) [reply facilities-mock-services/real-dependency-service-requests]: lol

File: facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    mock.when(
+            addQuery(
+                "/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VBA_Facilities/FeatureServer/0/query"
+                    + "?f=json&inSR=4326&outSR=4326&orderByFields=Facility_Number"
+                    + "&outFields=*&resultOffset=0&returnCountOnly=false"
+                    + "&returnDistinctValues=false&returnGeometry=true&where=1=1"))
+        .respond(
+            response()
+                .withStatusCode(200)
+                .withHeader(contentTextPlain())
```
Comment (jblefkove-va) [inline facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java]: json?

Comment (jblefkove-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java]: I fix!

Comment (bschofield-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java]: Actually, the mistake is on the other calls.
```
$ curl -s -D /dev/stdout "https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VBA_Facilities/FeatureServer/0/query?f=json&inSR=4326&outSR=4326&orderByFields=Facility_Number&outFields=*&resultOffset=0&returnCountOnly=false&returnDistinctValues=false&returnGeometry=true&where=1=1" -o /dev/null -H Accept:application/json
HTTP/2 200
content-type: text/plain; charset=utf-8
```
arcgis returns `text/plain` for _everything_. ... esri.

Comment (bschofield-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java]: i unfix!

Comment (jblefkove-va) [reply facilities-mock-services/src/main/java/gov/va/api/lighthouse/facilitiesmockservices/MockServices.java]: image

**Pull request merged**